src/app/components/ImageWithCaption/index.tsx

```diff
@@ -0,0 +1,119 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext } from 'react';
+import buildIChefURL from '../../lib/utilities/ichefURL';
+import urlWithPageAnchor from '../../lib/utilities/pageAnchor';
+import { createSrcsets } from '../../lib/utilities/srcSet';
+import filterForBlockType from '../../lib/utilities/blockHandlers';
+import { Services } from '../../models/types/global';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import Copyright from '../Copyright';
+import Caption from '../../legacy/containers/Caption';
+import IMAGE from '../Image';
```

Comment:
Is there a reason why the component name is all upper case here?

Reply:
Looking again, there isn't really a reason. I followed what looked like a pattern of uppercasing the new Image component, e.g: https://github.com/bbc/simorgh/blob/50605ba323c1d36507a5245257cfc052d2ccd89a/src/app/legacy/components/OptimoPromos/Image/index.jsx

But the reason for that in that example is because the exported component is also called `Image`, so the imported image component needed uppercased so that it didn't need to be called something else.

I've changed this now to normal casing.